app/assets/stylesheets/custom-bootstrap.css.scss

```diff
@@ -34,11 +34,8 @@ th.text-center {
   text-overflow: ellipsis;
 }
 
-.remove_top_padding {
-  margin-top:-20px;
-}
 .control-label-required {
-	@extend .control-label;
+	@extend .control-label !optional;
```

Comment:
It looks like the `control-label` class may have been removed in bootstrap4, so this extend may just no longer be necessary: https://getbootstrap.com/docs/4.0/migration/#forms-1